www/pages/guides/now.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Deploy on Now'
+menu: side
+linkheadings: 3
+index: 5
+---
+
+## Deploying your site on Now
+
+Requires an account with [Vercel](https://vercel.com/) linked to your GitHub account. If you need to sign up, the process is simple as is the deployment.
+
+After you have created your Greenwood project and pushed your code to a GitHub repo...
+
+- Log in to your Vercel account and proceed to the dashboard.
```

Comment:
Sorry, should have caught this before, but maybe for more procedural / step-by-step guide like this, an ordered / numbered list would be more appropriate?  It would also allow someone to call out a specific step by number if they needed to reference a specific action.  (or at least make it easier to do so)